Backend/src/constants.json

```
@@ -0,0 +1,9 @@
+{
+    "CAR_PORT": 4003,
+    "CAR_SERVER": "localhost",
+    "pi server to replace car server": "192.168.1.15",
```

The name is a bit verbose.
```suggestion
    "PI_ADDRESS": "192.168.1.15",
```